README.md

```diff
@@ -239,7 +239,9 @@ By passing `-t lcov` you could generate an lcov.info file and pass it to genhtml
 genhtml -o ./target/debug/coverage/ --show-details --highlight --ignore-errors source --legend ./target/debug/lcov.info
 ```
 
-#### Coveralls/Codecov output
+LCOV output should be used when uploading to Codecov, with the `--branch` argument for partial coverage support.
```

Comment:
Wdym by "for partial coverage support"?

Reply:
Partial coverage means the line is only partially covered. For example if you have an if statement which is short circuited:

`if(a | b)` would only be partially covered if the tests only ever run it for `a == true`

Do you have any suggestions for clearer wording?

Reply:
`partial coverage support` sounds like the coverage support is partial. That is, not fully implemented.

What about simply `branch coverage support`?

Reply:
Another option could be `to enable partial line coverage`.

Reply:
That makes sense. Technically codecov displays if the line is partially covered, which is computed from branch coverage, but the effect is the same. I think you're right; just calling it branch coverage is clearer